[IMP] Subscriptions: Reports update (#10878)
odoo/documentation: content/applications/sales/subscriptions/reports.rst

.. |MRR| replace:: :abbr:`MRR (Monthly Recurring Revenue)`
.. |ARR| replace:: :abbr:`ARR (Annual Recurring Revenue)`

The Odoo **Subscriptions** app provides a series of reporting pages to help companies analyze how
subscriptions are performing.

On the *Subscriptions Analysis* reporting page, users can view specific data related to recurring
subscriptions, quantity of subscriptions, in-progress or paused subscriptions, and more.

The *Retention Analysis* reporting page provides an organized table of subscription retention
percentages over any period of time.

The *MRR Breakdown* reporting page clearly divides |MRR| and |ARR| metrics for subscriptions into
various graphs, lists, and charts.

And the *MRR Analysis* reporting page offers a time-based collection of analytics showcasing how
subscription |MRR| and |ARR| have changed over the course of any given period of time.

Reporting page elements
=======================

All subscriptions-based reporting pages can be accessed via the :guilabel:`Reporting` header menu in
the **Subscriptions** app.

The following sections describe elements found on each reporting page.

Filters and Group By
--------------------

> Filters are used to narrow down metrics to show specific analytics, whereas groupings (via the

Reviewer: ```suggestion
*Filters* are used to narrow down metrics to show specific analytics, whereas groupings (via the
```

> *Group By* option) are used to gather specific sections' data into groups for more

Reviewer: ```suggestion
*Group By* option) are used to gather the data from specific sections into groups for more
```

This section refers to both filters and groupings, as a combination of the two can be saved in the
:guilabel:`Favorites` section.

To modify the results being shown on any reporting page, click the :icon:`fa-caret-down`
:guilabel:`(down arrow)` icon to the right of the search bar. Doing so reveals a drop-down menu of
detailed filter and grouping options.

.. note::
   Multiple filter options and grouping options can be selected at once.

.. figure:: reports/reporting-filter-options-dropdown.png
   :align: center
   :alt: The reporting filter drop-down menu of options in the Odoo Subscriptions application.

>    The reporting filter drop-down menu of options in the Odoo Subscriptions application.

Author: It might not be strictly necessary, but it was included when I was shown how to use figures by Felicia. I believe it might still be beneficial to people using assisted reading devices to have the alt text compared to only captions.

- :guilabel:`Flip axis`: the `x` and `y` axis of the pivot data table flip.
- :guilabel:`Expand all`: all the available rows and columns of the pivot data table expand fully.

> - :guilabel:`Download .xlsx`: the pivot data table is downloaded as an :file:`.xlsx`` file.

Reviewer 2: ```suggestion
- :guilabel:`Download .xlsx`: the pivot data table is downloaded as an :file:`.xlsx` file.
```

   :align: center
   :alt: The default view of the Subscriptions Analysis reporting page in Odoo Subscriptions.

When the :guilabel:`Measures` button on the :guilabel:`Subscriptions Analysis` page is

> clicked, a series of metric-related options become available as a drop-down menu.

Reviewer 2: ```suggestion
clicked, a series of metric-related options becomes available as a drop-down menu.
```

   :align: center
   :alt: The default view of the Subscriptions Analysis reporting page in Odoo Subscriptions.

When the :guilabel:`Measures` button on the :guilabel:`Subscriptions Analysis` page is

> clicked, a series of metric-related options become available as a drop-down menu.

Reviewer: ```suggestion
clicked, a series of metric-related options becomes available as a drop-down menu.
```

By default, the data displayed on the :guilabel:`MRR Breakdown` reporting page is in graph view,

> with the :guilabel:`Bar Chart` option and the :guilabel:`Stacked` option selected.

Reviewer: ```suggestion
with the :guilabel:`Bar Chart` option and :guilabel:`Stacked` option selected.
```

.. tip::
   To make a different measure option the default, first, select the desired measure from the
   :guilabel:`Measures` drop-down menu. Then, click the :icon:`fa-caret-down` :guilabel:`(down
   arrow)` icon in the search bar to open the mega menu of filters and groupings.

   In the :guilabel:`Favorites` column, click the :icon:`fa-caret-down` :guilabel:`(down arrow)`
   icon beside :guilabel:`Save current search` to reveal a field, where a title can be entered,
   along with two checkboxes::guilabel:`Default filter` and :guilabel:`Shared`.

>    Tick the box for :guilabel:`Default filter`, and click :guilabel:`Save`.

Reviewer: ```suggestion
   Tick the checkbox for :guilabel:`Default filter`, and click :guilabel:`Save`.
```
